bot.py

```diff
@@ -152,86 +228,39 @@ async def _reply(self, ctx, content, anonymous=False):
                             'discriminator': ctx.author.discriminator,
                             'avatar_url': str(ctx.author.avatar_url_as(static_format='png', size=1024)),
                             'mod': True,
-                        },
-                        'attachments': attachments,
+                        }  # ,
+                        #'attachments': attachments,
                     }
                 }
             },
         )
 
-        embed = discord.Embed(title='Moderator message', description=content, color=0x7ED321)
-        if not anonymous:
-            embed.set_author(name=f'{ctx.author} ({ctx.author.id})', icon_url=ctx.author.avatar_url)
-
-        else:
-            embed.title = '[ANON] Moderator message'
-            embed.set_author(
-                name=f'{ctx.author} ({ctx.author.id}) as r/NintendoSwitch',
-                icon_url='https://cdn.mattbsg.xyz/rns/snoo.png',
+    @cog_ext.cog_slash(
+        name='open',
+        guild_ids=guildList,
+        description='Open a modmail thread with a user, using your username',
+        permissions={config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]},
```

Comment:
`{config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]}` should be made into a static variable and passed instead.